oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-root.json

```diff
@@ -51,6 +51,29 @@
               "unit": "%"
             }
           ]
+        },
+        {
+          "x": 0,
+          "y": 0,
+          "w": 24,
+          "h": 2,
+          "i": "100",
+          "type": "Text",
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
```

Comment:
Are we changing all links to blue? Some were black only.

Reply:
I changed all links to blue, then users could notice it's a link.